ui/app/src/components/CreateFolderDialog/CreateFolderContainer.tsx

```diff
@@ -242,14 +272,31 @@ export function CreateFolderContainer(props: CreateFolderContainerProps) {
 						}}
 						onChange={(event) => onInputChanges(applyFolderNameRules(event.target.value, { allowBraces }))}
 					/>
+					{rename && containsItemsInWorkflow && (
+						<Alert severity="warning" icon={false} sx={{ mb: 1 }}>
+							<FormControlLabel
+								disableTypography
+								control={<Checkbox onChange={onCancelPackagesAckChange} />}
+								label={
+									<Typography>
+										<FormattedMessage defaultMessage="The item is part of one or more publishing packages. Editing it will cancel the packages." />
+									</Typography>
+								}
+							/>
+						</Alert>
+					)}
 					{rename && <FolderMoveAlert initialExpanded checked={moveFolderAck} onChange={onMoveFolderAckChange} />}
 				</form>
 			</DialogBody>
 			<DialogFooter>
 				<SecondaryButton onClick={onCloseButtonClick} disabled={isSubmitting}>
 					<FormattedMessage id="words.cancel" defaultMessage="Cancel" />
 				</SecondaryButton>
-				<PrimaryButton onClick={onSubmit} disabled={isSubmitting || !isValid} loading={isSubmitting}>
+				<PrimaryButton
+					onClick={onSubmit}
+					disabled={isSubmitting || !isValid || fetchingAffectedPackages}
```

Comment:
```suggestion
					disabled={!isValid}
```

Reply:
We would lose the `isSubmitting` validation, should I keep it here or add it to the `isValid` var?

Reply:
isSubmitting is considered by the loading prop of the button